### PR TITLE
[DNM] [TEST ONLY] Disable block authoring backoff for all networks

### PR DESCRIPTION
**Pull request by s0me0ne-unkn0wn**
This is for testing purposes only. Do not merge.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4640780

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4641939

File: polkadot/node/overseer/src/lib.rs [hunk 1]
```diff
@@ -839,9 +839,6 @@ where
 			self.on_head_deactivated(deactivated)
 		}
 
-		self.broadcast_signal(OverseerSignal::BlockFinalized(block.hash, block.number))
```
Comment (sandreim) [inline polkadot/node/overseer/src/lib.rs:842]: I think we should also skip the above active leaves pruning.

Comment (s0me0ne-unkn0wn) [reply polkadot/node/overseer/src/lib.rs:842]: Would it be fair? I mean, don't we leak more memory this way than would be leaked due to backoff disabling only?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 3/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4642769

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4651102

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4651539

**Pull request closed without merging**